metabase/metabase
# fix(sdk): generated cli components has inconsistent styles after the style leak fix

Closes https://github.com/metabase/metabase/issues/49652

### Description

After we fixed the style leak issue in https://github.com/metabase/metabase/issues/44802, the generated CLI components looks really strange as we were relying heavily on the leaked styles (e.g. to apply the background color to the page's background). This PR restores the generated component's styles.

### How to verify

- Run the CLI locally. I usually use `docker rm -f metabase-enterprise-embedding && yarn add file:../metabase/resources/embedding-sdk && npx @metabase/embedding-sdk-react start` on an empty Vite project to run.
- The generated components should have a normal style again.

### Demo

Light Theme

![CleanShot 2567-11-08 at 01 08 38@2x](https://github.com/user-attachments/assets/3fb1dc2a-771f-4b9f-85f8-6a3f7d52e78d)

Dark Theme

![CleanShot 2567-11-08 at 01 08 46@2x](https://github.com/user-attachments/assets/3509a511-4768-4789-b737-200ee839ec52)
